src/4D_api.cpp

```diff
@@ -1389,6 +1389,10 @@ int proj_errno_set (const PJ *P, int err) {
     /* For P==0 err goes to the default context */
     proj_context_errno_set (pj_get_ctx ((PJ *) P), err);
     errno = err;
+
+    // pj_errno was also set by pj_c tx_set_errno(), left out by mistake here?
+    // proj_errno_reset also sets it.
+    pj_errno = err;
```

Comment:
do we still have users of pj_errno ? Couldn't we remove it completely ? Can be a follow up task

Reply:
I wasn't sure, I added it here because the two errno-setters behaved differently and I suspected something might break if `pj_errno` wasn't set. It is likely that it can be removed all together, @busstoptaktik has been working towards that for quite some time at least.

Reply:
Seems to make no difference if it's there or not. Removed.

Reply:
my point was about removing ``PROJ_DLL extern int pj_errno;`` completely

Reply:
I know, for now I just wanted to not increase the use of it further. I'll look into how difficult it is to remove altogether now. If is easy I'll do it, if not I'll add an issue about getting it done.

Reply:
Okay, it wasn't that complicated and seems to have gone okay. `pj_errno` removed in e7a66db.